### ISSUE TITLE
DataQualityFlag.query() suddenly throwing a bug

### ISSUE THREAD

**Issue by karamerfeld** [gwpy/gwpy#967]
I have a bit of code that was working last Thursday, and then I tried to run it again, and it is suddenly throwing an error.  Here is the code:

```
from gwpy.segments import DataQualityFlag as DQF 

flagL = DQF.query('L1:DMT-SCIENCE',1164556817, 1187733618 )
```

And here is the error:
```
AttributeErrorTraceback (most recent call last)
<ipython-input-6-e003f8346662> in <module>()
----> 1 flagL = DQF.query('L1:DMT-SCIENCE',1164556817, 1187733618 )
      2 flagH = DQF.query('H1:DMT-SCIENCE',1164556817, 1187733618 )
      3 

/home/detchar/opt/gwpysoft-2.7/lib/python2.7/site-packages/gwpy/segments/flag.pyc in query(cls, flag, *args, **kwargs)
    399         query_ = _select_query_method(
    400             cls, kwargs.get('url', DEFAULT_SEGMENT_SERVER))
--> 401         return query_(flag, *args, **kwargs)
    402 
    403     @classmethod

/home/detchar/opt/gwpysoft-2.7/lib/python2.7/site-packages/gwpy/segments/flag.pyc in query_dqsegdb(cls, flag, *args, **kwargs)
    497                 data, versions, _ = apicalls.dqsegdbCascadedQuery(
    498                     protocol, server, out.ifo, out.tag, request,
--> 499                     int(start), int(end))
    500                 data['metadata'] = versions[-1]['metadata']
    501             else:

/usr/lib/python2.7/site-packages/dqsegdb/apicalls.pyc in dqsegdbCascadedQuery(protocol, server, ifo, name, include_list_string, startTime, endTime, debug)
    515         print versionQueryURL
    516     try:
--> 517         versionResult=urifunctions.getDataUrllib2(versionQueryURL)
    518     except HTTPError as e:
    519         if e.code==404:

/usr/lib/python2.7/site-packages/dqsegdb/urifunctions.pyc in getDataUrllib2(url, timeout, logger, warnings)
    112     except urllib2.URLError,e:
    113         #print e.read()
--> 114         warnings.warn("Issue accesing url: %s; Reason: %s" % (url,str(e.reason)))
    115         try:
    116             type, value, traceback_stack = sys.exc_info()

AttributeError: 'bool' object has no attribute 'warn'
```

**Comment by alurban** [comment]
Hi @karamerfeld, this is likely because `segments.ligo.org` is down for maintenance at the moment. Can you try passing `url=https://segments-backup.ligo.org`, and see if that works? Note, this backup server may not have the most recent segments, so it may return an empty object.

**Comment by duncanmmacleod** [comment]
I am certain @alurban is correct, in any case the real bug was already fixed by ligovirgo/dqsegdb#29, but just hasn't been released yet. This is not a GWpy failure.
